### DISCUSSION TITLE
Does Auto-Type Support Entering Data Into Two Password Fields?

### DISCUSSION THREAD

**Discussion by 1artist** [keepassxreboot/keepassxc#9641] [Q&A] [answered]
I have the latest Keepassxc version running on my Win10 PC.  I am a new user and I hit a brick wall when I try to sign into my Public Library website using the Keepassxc Auto-Type feature.  In short, my Public Library website does not use a Username and Password field. Instead it uses a Last Name Field, a Library Card Number Field and a PIN Field.  Using the code shown below I am able to make Auto-Type enter data correctly into the Last Name Field, and the Library Card Number Field. However, how do I now modify the code below so I can make Auto-Type enter a 979745 PIN into in a PIN Field?

`{USERNAME}{TAB}{DELAY 1000}{PASSWORD}{DELAY 1500}{TAB}{DELAY 1000} (Need Help Here at the end)`

Any advice welcome.   Thank you for your time.

**Comment by droidmonkey** [comment, marked as answer]
Create an advanced attribute (go to the advanced tab) called `pin`. Then at the end of your Auto-Type sequence put `{S:pin}`

See all possible Auto-Type actions here: https://keepassxc.org/docs/KeePassXC_UserGuide#_auto_type_actions

**Comment by 1artist** [comment]
Droidmonkey, back from some business travel and I finally tested out your suggestion.  I am glad to say the Auto-Type sequence shown below works perfectly for what I need:

`{USERNAME}{TAB}{DELAY 1000}{PASSWORD}{TAB}{DELAY 1000}{S:pin}{DELAY 1000}{ENTER}`

The website's PIN request is handled flawlessly.   Thank you a lot for opening my eyes to how the Advanced tab works because that was the key to auto typing the PIN content. 

Regarding the link you posted.  I reviewed the same link before creating this post and unfortunately it did not help me figure out how the Advanced tab works.  In closing, does anybody in this forum know of any website that provides examples for all the bracketed commands shown in the link below?  For a Syncthing newbie like me, it would help me a lot if examples where provided that show the bracketed commands in use to execute a variety of tasks. 

[https://keepassxc.org/docs/KeePassXC_UserGuide#_auto_type_actions](url)
